### PR TITLE
665 handle table for products

### LINKED ISSUE #665: [Product] Update/handle table for products in converter

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #665 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/biozen-biologics-lc-columns
- After: https://665-handle-table-for-products--danaher-ls-aem--hlxsites.hlx.page/us/en/products/family/biozen-biologics-lc-columns

### CODE REVIEW

**Review by davenichols-DHLS — APPROVED** [review]
importer change, so presume we have to merge and then publish to see.

File: tools/importer/transformers/product.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ const render = (main, element, document) => {
       main.append(block);
       break;
     }
+    case 'table': {
+      const tableEL = element.querySelector('template')?.content?.firstElementChild;
```
Comment (duynguyen) [inline tools/importer/transformers/product.js:42]: is it better to do: `element.querySelector('template')?.content?.querySelector('table')` ?

File: tools/importer/transformers/util.js [hunk 2]
```diff
@@ -283,4 +291,8 @@ export const render = {
     const videoEl = item.content ? item.content.querySelector('div.video') : item;
     row.push(videoembed(videoEl, document));
   },
+
+  table: (item, row) => {
+    row.push(item.querySelector('template')?.content?.firstElementChild);
```
Comment (duynguyen) [inline tools/importer/transformers/util.js:296]: is it better to do: `item.querySelector('template')?.content?.querySelector('table')` ?

Comment (rgravitvl) [reply tools/importer/transformers/util.js]: Okay

File: tools/importer/transformers/product.js (hunk 1 above)
Comment (rgravitvl) [reply tools/importer/transformers/product.js]: Okay

**Pull request merged**